src/rospkg/os_detect.py

```diff
@@ -242,6 +242,24 @@ def get_codename(self):
         raise OsNotDetected('called in incorrect OS')
 
 
+class RoboStack(OsDetector):
+    """
+    Detect RoboStack.
+    """
+    def is_os(self):
+        return "ROS_OS_OVERRIDE" in os.environ and os.environ["ROS_OS_OVERRIDE"].startswith("robostack")
```

Comment:
should we add this env variable into our activation scripts?

Reply:
Check the vinca PR :)